src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor

```diff
@@ -1,28 +1,91 @@
 ﻿@namespace Bit.Client.Web.BlazorUI
 @inherits BitComponentBase
 
-<div @ref="RootElement"
-     style="@StyleBuilder.Value"
-     class="@ClassBuilder.Value">
-    <span>
-        <i class="bit-icon bit-icon--@MessageBarIconName"></i>
-    </span>
-    <span class="@((Truncated && !ElementTruncateState) || IsMultiline ? "not-truncate" : "")">
-        @ChildContent
-    </span>
-    <span>
-        @if (Truncated)
-        {
-            <span @onclick="ToggleElementTruncate">
-                <i class="bit-icon bit-icon--@(ElementTruncateState ? "DoubleChevronDown" : "DoubleChevronUp" )"></i>
-            </span>
-        }
-
-        @if (OnDismiss.HasDelegate)
-        {
-            <span @onclick="Dismiss">
-                <i class="bit-icon bit-icon--@DismissIconName"></i>
-            </span>
-        }
-    </span>
-</div>
+@if (IsMultiline)
+{
+    <div class=@($"{RootElementClass}--root")
```

Comment:
This class hasn't been used anywhere and also we are creating two root classes in BitComponentBase OnInitialized ClassBuilder 
               .Register(() => RootElementClass)
               .Register(() => $"{RootElementClass}-{VisualClassRegistrar()}")
  so this would make this class obsollete